components/centraldashboard/package.json

```diff
@@ -70,8 +69,7 @@
         "chart.js": "^2.8.0",
         "chartjs-plugin-crosshair": "^1.1.4",
         "express": "^4.17.1",
-        "node-fetch": "^2.6.0",
-        "web-animations-js": "^2.3.2"
```

Comment:
According to `caniuse` https://caniuse.com/?search=Web%20Animations%20API
we can drop it~